templates/hardwarelab/view_device.html

```diff
@@ -26,6 +26,7 @@
                 alertify.alert("You successfully reserved a {{device.name}}");
                 $('#reservations')
                 .replaceWith(data); // log the returned json to the console
+                location.reload()
```

Comment:
Same thing here!
  $('#reservations').replaceWith(data);   << this updates your data with rerendered data. You don't want to reload the page